JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1145,4 +1186,90 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(SEK)
 }
 
-# </editor-fold> HELPER FUNCTIONS BLOCK
\ No newline at end of file
+.descriptivesQQPlot <- function(dataset, options,  qqvar, levelName=NULL) {
+
+  if (options$descriptivesQQPlot && is.null(options$variables))
+    return(list(result=NULL))
+
+  descriptivesQQPlot <- list()
+
+  if (!is.null(qqvar)) {
+
+    #to put a subtitle if there is a split
+    if(!is.null(levelName))
+      descriptivesQQPlot$title <- paste0(options$splitby, ": ", levelName)
+    else
+      descriptivesQQPlot$title <- " "
```

Comment:
We still want a title here: the variable name; otherwise we don't know which plot belongs to which variable.
But if you set the title in the beginning already just remove this here.